src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Atom10FeedFormatter.cs

```diff
@@ -326,11 +305,11 @@ internal Task<TextSyndicationContent> ReadTextContentFromAsync(XmlReaderWrapper
                         break;
                     case Atom10Constants.UpdatedTag:
                         await reader.ReadStartElementAsync();
-                        result.LastUpdatedTime = dateParser(await reader.ReadStringAsync(), Atom10Constants.UpdatedTag, ns);
+                        result.LastUpdatedTime = DateTimeParser(await reader.ReadStringAsync(), Atom10Constants.UpdatedTag, ns);
```

Comment:
>DateTimeParser [](start = 49, length = 14)

Question: why do we want to support users using a custom DateTime parser?

Reply:
One of the big issues at least Microsoft internal customers had with the Syndication library was that various rss feeds used odd datetime formats that we weren't handling. It was a lot of work to be able to create a derived class with different behavior. There's always going to be someone with a feed with dates that we can't handle which means we need to make it easy for them to do custom parsing. We do a best effort by parsing the formats in the Atom and RSS specs.